OneginiExampleAppSwift/Interactors/StartupInteractor.swift

```diff
@@ -14,21 +14,16 @@
 // limitations under the License.
 
 import UIKit
+import OneginiSDKiOS
 
 protocol StartupInteractorProtocol {
-    func oneginiSDKStartup(completion: @escaping (Bool, AppError?) -> Void)
+    func oneginiSDKStartup(completion: @escaping (AppError?) -> Void)
```

Comment:
no result needed?

Reply:
Nope, we use `func start(completion: @escaping (Error?) -> Void)` that returns only optional error so this Example app internal method only mimics that.